tools/topology/topology2/cavs-mixin-mixout-efx-hda.conf

```diff
@@ -80,7 +80,7 @@ Object.Pipeline {
 		}
 	]
 
-	passthrough-be [
+	be-dai [
```

Comment:
Sorry @ranj063 I find the naming confusing

a) a DAI can be bidirectional - this isn't
b) This is really a BE copier, there's nothing related to the DAI configuration, is there?

Reply:
a) yes this is birectional. its simply a BE DAI copier. The reason why I call this a be-dai instead of a be-copier is because it has a pipeline widget as well, not just a copier.
b) It's a copier alright and the DAI config for the copier comes from the type of DAI and is added at the time of instantiation.

But having said this, I really dont have a preference. I'll go with be-copier if you prefer that. I just dont think that's accurate either